piq/functional/base.py

```diff
@@ -84,3 +84,16 @@ def pow_for_complex(base: torch.Tensor, exp: Union[int, float]) -> torch.Tensor:
     x_real_pow = x_complex_pow_r * torch.cos(x_complex_pow_phi)
     x_imag_pow = x_complex_pow_r * torch.sin(x_complex_pow_phi)
     return torch.stack((x_real_pow, x_imag_pow), dim=-1)
+
+
+def crop_patches(x: torch.Tensor, size=64, stride=32) -> torch.Tensor:
```

Comment:
What happens if size < x[-1] (or x[-2])?  
I think we need a test to cover this case.